pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    nessie: Test run against a Nessie container
```

Comment:
nit: `localserver`, `nessieserver` or the like? just `nessie` seems confusing since every test is about (py)nessie